enterprise/api/v1/serializers.py

```diff
@@ -606,6 +648,21 @@ def validate(self, data):  # pylint: disable=arguments-differ
             if 'email' in data and not bool(data['email']):
                 raise serializers.ValidationError({'email': ['This field is required']})
 
+        # validate that enterprise customer catalogs exist in the system.
+        enterprise_catalogs = data.get('enterprise_catalogs')
+        enterprise_customer = data['enterprise_customer']
```

Comment:
Will this ever return `None`? Just want to make sure the query below to fetch enterprise catalogs doesn't throw any exception.

Reply:
`enterprise_customer` field is required by default (ref: https://github.com/edx/edx-enterprise/blob/master/enterprise/models.py#L2342) so it can not be `None`.